openreview/venue_request/venue_request.py

```diff
@@ -602,7 +602,7 @@ def setup_request_form(self):
                 'value-regex': '.*',
                 'order': 9
             },
-            'Abstract Registration Deadline': {
+            'abstract_registration_deadline': {
```

Comment:
We may need to rename the current notes with the old field name. I don't think there are too many.